src/lib.rs

```diff
@@ -157,8 +159,19 @@ impl AbsInfo {
 
 #[derive(Clone, Debug, PartialEq)]
 pub struct TimeVal {
-   pub tv_sec: c_long,
-   pub tv_usec: c_long,
+    pub tv_sec: c_long,
```

Comment:
I didn't make this formatting change, I think it was `cargo fmt`. If the maintainer of this repo is open to it, I'd be happy to submit the full set of `cargo fmt` changes as a separate PR.

Reply:
I would appreciate that. Thanks